samples/mixed/consumer/src/main.rs

```diff
@@ -158,7 +189,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let show_notification_command_provider_endpoint_info =
```

Comment:
Neat: the repeated `discover_digital_twin_provider_using_ibeji()` can be reduced with [partial!](https://docs.rs/partial_application/latest/partial_application/#:~:text=The%20partial%21%20macro%20allows%20for%20partial%20application%20of,move%20in%20front%20of%20the%20function%3A%20partial%21%20%28move..%29).
As well as the `send_subscribe_request()`